src/main/java/com/getindata/connectors/http/HttpPostRequestCallback.java

```diff
@@ -25,5 +25,5 @@ void call(
         RequestT requestEntry,
         String endpointUrl,
         Map<String, String> headerMap
-    );
+    ) throws PostRequestCallbackException;
```

Comment:
is there a backwards compatibility issue - as you are changing a method that existing user code could be using?

Reply:
Hey 👋 , I believe this is backwards compatible as classes still implement the interface even if they don't declare `throws PostRequestCallbackException `, but am no Java expert.